packages/next/client/image.tsx

```diff
@@ -697,54 +756,132 @@ export default function Image({
   useEffect(() => {
     handleLoading(imgRef, srcString, layout, placeholder, onLoadingCompleteRef)
   }, [srcString, layout, placeholder, isVisible])
-
+  const imgElementArgs = {
+    lazy: isLazy,
```

Comment:
Nick Pick: any reason you're using `lazy` instead of `isLazy`?

Reply:
changed to isLazy